doc/source/user_guide/io.rst

```diff
@@ -3880,6 +3880,19 @@ The look and feel of Excel worksheets created from pandas can be modified using
 * ``float_format`` : Format string for floating point numbers (default ``None``).
 * ``freeze_panes`` : A tuple of two integers representing the bottommost row and rightmost column to freeze. Each of these parameters is one-based, so (1, 1) will freeze the first row and first column (default ``None``).
 
+The styling and formatting of a worksheet's header and body can be easily customized using CSS by passing a dictionary object to
+``pd.io.formats.excel.ExcelFormatter.header_style`` and ``pd.io.formats.excel.ExcelFormatter.body_style``.
+
+.. code-block:: python
+
+    # Set Excel worksheet header and body style
+    >>> df = pd.DataFrame([{"A": 1, "B": 2, "C": 3}, {"A": 1, "B": 2, "C": 3}])
+    >>> pd.io.formats.excel.ExcelFormatter.header_style = {"font": {"bold": True},}
```

Comment:
I don't think this is doing what you might expect. In particular, this is not using getters/setters, but rather just overwriting a property.

```
class Foo:
    @property
    def foo(self):
        raise ValueError
        
    @foo.setter
    def foo(self, other):
        raise ValueError
        
print(Foo.foo)
# <property object at 0x7f5438093290>
Foo.foo = 5
print(Foo.foo)
# 5
```

Reply:
@rhshadrach @attack68 

What do we think would be the appropriate implementation for this request then? It seems like the [initial suggestion](https://github.com/pandas-dev/pandas/issues/52369) is not ideal.

Reply:
@rhshadrach @attack68 

What do we think would be the appropriate implementation for this request then? It seems like the [initial suggestion](https://github.com/pandas-dev/pandas/issues/52369) is not ideal.